src/modules/ModuleStaking/types.ts

```diff
@@ -18,6 +18,7 @@ export interface Pool {
   }
   staked: BigNumber
   earned: BigNumber
+  stakeTokenPrice: BigNumber
   createdAtBlock: number
   totalStaked: BigNumber
   annualPercentageRate: BigNumber
```

Comment:
All this `BigNumber`s need better typing. What units they represent? `Wei`, `WeiAsToken`, or what?

Reply:
Yes, you're right. I was doing this even before the last changes with Wei typing

Reply:
But price isn't a ```Wei``` or ```WeiAsToken```, so I'll leave BigNumber for it

Reply:
Ok, but what is the price?

Reply:
price of 1 WeiAsToken in USD

Reply:
Done